### ISSUE TITLE
QAT模型未更新

### ISSUE THREAD

**Issue by xingyueye** [alibaba/MNN#1784]
你好，我使用MNN的QAT训练时，发现模型好像并未更新，几次test结果都不变。而且放大学习率之后也是一样的情况
代码直接使用的example的代码，唯一的改动如下：
      `predict = net.forward(data)
        predict = F.sigmoid(predict)   # 网络最后一层为fc，加入一个sigmoid来与label计算交叉熵
        target = F.one_hot(F.cast(label, F.int), num_classes, 1, 0)
        loss = nn.loss.cross_entropy(predict, target)`

请教下这可能是什么原因导致的错误

**Comment by yyfcc17** [comment]
现在MNN的训练框架并不成熟，建议使用MNNPythonOfflineQuant进行量化，或者使用MNNConvert进行仅权值量化

**Comment by xingyueye** [comment]
非常感谢这么快速的回复🙏
还有个小问题，现在的 F.cross_entropy 是纯粹的交叉熵计算，而不像是pytorch还包含了 softmax计算？
这样的话怎么可以来和Pytorch进行匹配呢，MNN.expr 中只有sigmoid的实现，没有softmax

**Comment by yyfcc17** [comment]
最新python包是有softmax的

**Comment by yyfcc17** [comment]
最新python包是有softmax的
